optimum/utils/input_generators.py

```diff
@@ -575,3 +578,31 @@ def generate(self, input_name: str, framework: str = "pt"):
                 max_value=1,
                 framework=framework,
             )
+
+
+class DummyTimestepInputGenerator(DummyInputGenerator):
+    """
+    Generates dummy time step inputs.
+    """
+
+    SUPPORTED_INPUT_NAMES = ("timestep",)
+
+    def __init__(
+        self,
+        task: str,
+        normalized_config: NormalizedConfig,
+        batch_size: int = DEFAULT_DUMMY_SHAPES["batch_size"],
+        random_batch_size_range: Optional[Tuple[int, int]] = None,
+    ):
+        self.task = task
+        self.vocab_size = getattr(normalized_config, "vocab_size", 49408)
```

Comment:
I suggested a possible solution for this above.

Reply:
Good catch ! We don't need to set a default value there, I will remove it